products/cloudbuild/api.yaml

```diff
@@ -0,0 +1,181 @@
+          Path, from the source root, to a file whose contents is used for the template.
+      - !ruby/object:Api::Type::Array
+        name: 'ignoredFiles'
+        item_type: Api::Type::String
+        input: true
+        description: |
+          ignoredFiles and includedFiles are file glob matches using http://godoc/pkg/path/filepath#Match
+          extended with support for "**".
+
+          If ignoredFiles and changed files are both empty, then they are not
+          used to determine whether or not to trigger a build.
+
+          If ignoredFiles is not empty, then we ignore any files that match any
+          of the ignored_file globs. If the change has no files that are outside
+          of the ignoredFiles globs, then we do not trigger a build.
+      - !ruby/object:Api::Type::Array
+        name: 'includedFiles'
+        item_type: Api::Type::String
+        input: true
+        description: |
+          If any of the files altered in the commit pass the ignoredFiles filter
+          and includedFiles is empty, then as far as this filter is concerned, we
+          should trigger the build.
+
+          If any of the files altered in the commit pass the ignoredFiles filter
+          and includedFiles is not empty, then we make sure that at least one of
+          those files matches a includedFiles glob. If not, then we do not trigger
+          a build.
+      - !ruby/object:Api::Type::NestedObject
+        name: 'triggerTemplate'
+        description: |
+          Template describing the types of source changes to trigger a build.
+
+          Branch and tag names in trigger templates are interpreted as regular
+          expressions. Any branch or tag change that matches that regular
+          expression will trigger a build.
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'projectId'
+            description: |
+              ID of the project that owns the Cloud Source Repository. If
+              omitted, the project ID requesting the build is assumed.
+          - !ruby/object:Api::Type::String
+            name: 'repoName'
+            default_value: 'default'
+            description: |
+              Name of the Cloud Source Repository. If omitted, the name "default" is assumed.
+          - !ruby/object:Api::Type::String
+            name: 'dir'
+            description: |
+              Directory, relative to the source root, in which to run the build.
+
+              This must be a relative path. If a step's dir is specified and
+              is an absolute path, this value is ignored for that step's
+              execution.
+          - !ruby/object:Api::Type::String
+            name: 'branchName'
+            description: |
+              Name of the branch to build.
+          - !ruby/object:Api::Type::String
+            name: 'tagName'
+            description: |
+              Name of the tag to build.
+          - !ruby/object:Api::Type::String
```

Comment:
I'm not sure why the field is ever useful- it will build the same thing every time- but I prefer deferring to the product over excluding it and adding it back later. Also it's present in Terraform already and non-deprecated so we need to keep it regardless.